cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/macros/financial-item-flyout.html

```diff
@@ -26,19 +26,20 @@
 
    ========================================================================== #}
 
-{% macro expandable(value) %}
-<div class="o-expandable
-            o-expandable__padded">
-    <button class="o-expandable_header
+{% macro flyout(value) %}
+<div class="o-costs-group" data-js-hook="behavior_flyout-menu">
+    <button class="o-costs-group_header
+                   o-costs-group_target
```

Comment:
`o-expandable_target` is gone. You shouldn't need both `o-costs-group_header` and `o-costs-group_target`. Just merge them together into `o-costs-group_header`.

Reply:
👍